.circleci/config.yml

```
@@ -2,7 +2,7 @@ version: 2
 jobs:
   validate:
     docker:
-      - image: trussworks/circleci-docker-primary:tf12-4c1fd54273446484259597ae3da9deb2806498ed
+      - image: trussworks/circleci-docker-primary:tf12-e66fbea875bcb788b29b1b5f59142e8231961ec5
```

We've moved tf12 back into the base image, so you can remove the tf-12 bit
```suggestion
      - image: trussworks/circleci-docker-primary:e66fbea875bcb788b29b1b5f59142e8231961ec5
```